backend/app/app.go

```
@@ -53,22 +54,47 @@
 	}
 	app.tokenConfig = tokenConfig
 
-	signin, err := usecase.NewSignIn(pool, tokenConfig)
+	signin, err := handlers.NewSignInHandler(pool, tokenConfig)
 	if err != nil {
 		app.Shutdown(context.Background())
```

golangci-lint (1.62.2)

Error return value of `app.Shutdown` is not checked

(errcheck)